docs/cart-and-checkout-blocks/slot-fills.md

```diff
@@ -6,7 +6,7 @@ tags: reference
 
 ## The problem
 
-You added custom data to the [Store API](https://github.com/woocommerce/woocommerce/blob/1675c63bba94c59703f57c7ef06e7deff8fd6bba/plugins/woocommerce-blocks/docs/third-party-developers/extensibility/rest-api/extend-rest-api-add-data.md). You changed several strings using [Checkout filters](./available-filters/README.md). Now you want to render your own components in specific places in the Cart and Checkout.
+You added custom data to the [Store API](https://github.com/woocommerce/woocommerce/blob/1675c63bba94c59703f57c7ef06e7deff8fd6bba/plugins/woocommerce/client/blocks/docs/third-party-developers/extensibility/rest-api/extend-rest-api-add-data.md). You changed several strings using [Checkout filters](./available-filters/README.md). Now you want to render your own components in specific places in the Cart and Checkout.
```

Comment:
I just realized these aren't going to work, since at that commit the new structure was not available. The commit-specific references should stay unchanged.

Reply:
Restored in https://github.com/woocommerce/woocommerce/pull/54971/commits/1e6ff409d4571b3a2509d9f339dfb63cbef70563